lib/assets/vega_lite/main.js

```diff
@@ -7,7 +7,82 @@ const DEFAULT_DATASET_NAME = "source_0";
 
 const throttledResize = throttle((view) => view.resize(), 1_000);
 
+const markColor = "#6583ff";
+const mainColors = ["#6583ff", "#ffa83f", "#e2474d", "#4aa148", "#2d4cdb", "#b7641f", "#bc1227", "#137518"];
+const blues = ["#b2c1ff", "#8ba2ff", "#6583ff", "#3E64ff", "#2d4cdb", "#1f37b7", "#132593"];
+const lightGray = "#e1e8f0";
+const mediumGray = "#445668";
+const darkGray = "#1c2a3a";
+
+const liveBookTheme = {
+  background: "#fff",
+
+  title: {
+    anchor: "center",
+    fontSize: 18,
+    fontWeight: 400,
+    color: mediumGray,
+    fontFamily: "Inter",
+    font: "Inter",
+  },
+
+  arc: {fill: markColor},
+  area: {fill: markColor},
+  line: {stroke: markColor, strokeWidth: 2},
+  path: {stroke: markColor},
+  rect: {fill: markColor},
+  shape: {stroke: markColor},
+  symbol: {fill: markColor, strokeWidth: 1.5, size: 50},
+  point: {stroke: markColor},
+  bar: {fill: markColor, stroke: null},
+  circle: {fill: markColor},
+  tick: {fill: markColor},
```

Comment:
We may be missing some marks here, like `rule`. Some marks like `boxplot` are composed of base marks, so maybe the theme will apply, but please check :)

Sidenote: there is `Vl.from_json("...")`, which makes it simple to try examples from vega-lite docs.